pkg/tecdsa/dkg/gen/pb/message.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Let's merge `main` back to this branch once #3273 lands and regenerate this file to make sure we use the right proto version for that. In #3257 we updated the protobuf and now `make` should be used for generating files.

Same comment to all other `*.pb.go` files in this PR.

Reply:
This was generated using the new way presented in https://github.com/keep-network/keep-core/pull/3257

Reply:
I confirmed it locally by merging `main` to this branch and re-generating. No differences.